pkg/container/qemu_runner.go

```diff
@@ -1222,30 +1231,25 @@ func getAvailableMemoryKB() int {
 		for s.Scan() {
 			var n int
 			if nItems, _ := fmt.Sscanf(s.Text(), "MemTotal: %d kB", &n); nItems == 1 {
-				return n
+				return int64(n), nil
 			}
 		}
 	case "darwin":
-		cmd := exec.Command("sysctl", "hw.memsize")
+		cmd := exec.Command("sysctl", []string{"-n", "hw.memsize"}...)
```

Comment:
Good call.

After a bit of digging around, it looks like `hw.memsize_usable` is a separate field if we're interested in using this:
```
hw.memsize_usable: 37984092160
```

Reply:
my bad left changes that are superseded by https://github.com/chainguard-dev/melange/pull/2005#pullrequestreview-2862524419

removed them